modules/webp/webp_common.cpp

```diff
@@ -97,6 +101,7 @@ Vector<uint8_t> _webp_lossless_pack(const Ref<Image> &p_image) {
 
 	WebPMemoryWriter wrt;
 	config.exact = 1;
+	config.near_lossless = p_near_lossless_quality * 100.0f;
```

Comment:
I wonder what this means?

Reply:
`config` is used by libwebp to define some parameters. Whenever the lossless workflow is used, the value of `config.near_lossless` is set to 100, which means no loss. Any value below 100 triggers near lossless, so the quality from lossy is provided to near_lossless.